Formula/libthemis.rb

```diff
@@ -1,34 +1,34 @@
 class Libthemis < Formula
-  desc 'High-level cryptographic primitives'
-  homepage 'https://www.cossacklabs.com/themis'
-  head 'https://github.com/cossacklabs/themis.git'
-  url 'https://github.com/cossacklabs/themis/archive/0.14.0.tar.gz'
-  sha256 '2efb793e0ef604fb97258b07671a83135ad9229d83b92d7758b43510dcc6cb07'
-  version '0.14.0'
+  desc "High-level cryptographic primitives"
+  homepage "https://www.cossacklabs.com/themis"
+  url "https://github.com/cossacklabs/themis/archive/0.15.1.tar.gz"
+  head "https://github.com/cossacklabs/themis.git"
+  version "0.15.1"
+  sha256 "0bd25db4c48d25031926f9700718a1bf8807bb60755a97bf9fcd60492f491d0d"
   revision 0
 
-  depends_on 'openssl@1.1'
+  depends_on "openssl@3"
 
-  option 'with-cpp', 'Install C++ header files for ThemisPP'
-  option 'with-java', 'Install JNI library for JavaThemis'
+  option "with-cpp", "Install C++ header files for ThemisPP"
+  option "with-java", "Install JNI library for JavaThemis"
 
   def install
-    ENV['ENGINE'] = 'openssl'
-    ENV['ENGINE_INCLUDE_PATH'] = Formula['openssl@1.1'].include
-    ENV['ENGINE_LIB_PATH'] = Formula['openssl@1.1'].lib
-    ENV['PREFIX'] = prefix
-    system 'make', 'install'
-    if build.with? 'cpp'
-      system 'make', 'themispp_install'
+    ENV["ENGINE"] = "openssl"
```

Comment:
any particular reason why ruby prefers `"` to `'`? 🤔

Reply:
The same question. There are many discussions above the question, but in Ruby we use `"` when we expecting string interpolation, and `'` when we don't. So, using `"` in all places brings potential risks to get interpolation where we don't expect it. I wouldn't suggest such approach.

Reply:
@radetsky let's switch back to the `'`?

Reply:
Ok.